### PR TITLE
GraphQL NFT query with parmeter

### PR DESCRIPTION
**Pull request by echohtp**
Adds graphql queries for two new objects from postgres

-`Nfts(update_authority: [String,...], creators [String,...])`
-`Nft(address: String)`

### CODE REVIEW

**Review by kespinola — APPROVED** [review]

**Comment by echohtp** [comment]
not ready for merge yet, going to add the code to query nfts by creator address

File: crates/graphql/src/main.rs [hunk 1]
```diff
@@ -15,7 +15,7 @@ use actix_web::{middleware, web, App, Error, HttpResponse, HttpServer};
 use indexer_core::{clap, clap::Parser, db, ServerOpts};
 use juniper::http::{graphiql::graphiql_source, GraphQLRequest};
 
-use crate::schema::Schema;
+use crate::{db::Pool, schema::Schema};
```
Comment (ray-kast) [inline crates/graphql/src/main.rs:18]:
```suggestion
use crate::schema::Schema;
```

File: crates/graphql/src/main.rs [hunk 2]
```diff
@@ -25,6 +25,9 @@ struct Opts {
     server: ServerOpts,
 }
 
+// To make our context usable by Juniper, we have to implement a marker trait.
+// impl juniper::Context for Context {}
+
```
Comment (ray-kast) [inline crates/graphql/src/main.rs:30]:
```suggestion
```

File: crates/graphql/src/main.rs [hunk 3]
```diff
@@ -58,14 +61,15 @@ fn main() {
         } = Opts::parse();
 
         // TODO
```
Comment (ray-kast) [inline crates/graphql/src/main.rs:63]:
```suggestion
```

File: crates/graphql/src/schema.rs [hunk 4]
```diff
@@ -60,6 +105,10 @@ impl MutationRoot {
...
 
-pub fn create() -> Schema {
-    Schema::new(QueryRoot {}, MutationRoot {}, EmptySubscription::new())
+pub fn create(db: Pool) -> Schema {
+    Schema::new(QueryRoot { db }, MutationRoot {}, EmptySubscription::new())
 }
+
+// pub fn getNFT(){
+
+// }
```
Comment (ray-kast) [inline crates/graphql/src/schema.rs:114]:
```suggestion
```

**Review by ray-kast — CHANGES_REQUESTED** [review]

**Review by ray-kast — APPROVED** [review]
:+1: :smile:

**Pull request merged**